ui/package.json

```diff
@@ -21,7 +21,9 @@
     "@types/react": "^17.0.1",
     "@types/react-dom": "^17.0.1",
     "axios": "^0.25.0",
+    "buffer": "^6.0.3",
     "classnames": "^2.2.6",
+    "crypto": "npm:crypto-browserify@^3.12.0",
```

Comment:
This is just an alias syntax, installing the `crypto-browserify` module from `npm` as `crypto` (the name of the built-in node module it’s polyfilling).